app/models/import/import_external_activity.rb

```diff
@@ -54,4 +56,15 @@ def error(job, exception)
     import.update_attribute(:job_finished_at, Time.current)
     import.update_attribute(:progress, -1)
   end
+
+  def add_new_admin_tag(import_activity, tag_type, tag_list)
```

Comment:
Can you change the name to `add_new_admin_tags`? 
Since it takes a array of tags, that will make it more clear.